apps/coscrad-frontend/src/components/image-in-content/image-in-content.tsx

```diff
@@ -0,0 +1,45 @@
+export type ImageAsset = {
+    title: string;
+    src: string;
+    width: number;
+    height: number;
+};
+
+interface ImageInContentProps {
+    image: ImageAsset;
+    displayWidth?: string;
+    title?: string;
+    alt?: string;
```

Comment:
`alt` should **never** be optional. It's an accessibility issue.  We actually have a lint rule that enforces this and I fear you may be bypassing it with this construction.

Reply:
This component and its interfaces are just a sketch of what's to come.  For now I've modified the `ImageInContent` component to automatically generate the `alt` tag from the image title instead of having `alt` as a prop.